ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/health/HealthService.scala

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2019 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.platform.sandbox.health
+
+import akka.Done
+import akka.stream.Materializer
+import akka.stream.scaladsl.{Sink, Source}
+import com.digitalasset.grpc.adapter.utils.DirectExecutionContext
+import com.digitalasset.grpc.{GrpcException, GrpcStatus}
+import com.digitalasset.platform.api.grpc.GrpcApiService
+import io.grpc.health.v1.health.{HealthCheckRequest, HealthCheckResponse, HealthGrpc}
+import io.grpc.stub.StreamObserver
+import io.grpc.{Context, ServerServiceDefinition, Status}
+
+import scala.concurrent.duration.{DurationInt, FiniteDuration}
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.control.NonFatal
+import scala.util.{Failure, Success}
+
+class HealthService(watchThrottleFrequency: FiniteDuration = 1.second)(
+    implicit materializer: Materializer,
+    executionContext: ExecutionContext,
+) extends HealthGrpc.Health
+    with GrpcApiService {
+  private val servingResponse = HealthCheckResponse(HealthCheckResponse.ServingStatus.SERVING)
+
+  override def bindService(): ServerServiceDefinition =
+    HealthGrpc.bindService(this, DirectExecutionContext)
+
+  override def close(): Unit = ()
+
+  override def check(request: HealthCheckRequest): Future[HealthCheckResponse] =
+    Future.successful(servingResponse)
+
+  override def watch(
```

Comment:
Thanks. If you add `"@com_github_grpc_grpc//src/proto/grpc/health/v1:health_proto_descriptor",` to `//ledger/ledger-api-akka:ledger-api-akka-srcs` [0], you can implement the `HealthAkkaGrpc` trait (c.f. `GrpcTransactionService` [1]) to only have to implement the generated abstract method `watchSource` without having to mess around with the stream observer.

[0] https://github.com/digital-asset/daml/blob/9f9d2d0458013cb8f75dfc5c630a1b711c38359a/ledger/ledger-api-akka/BUILD.bazel#L7-L9
[1] https://github.com/digital-asset/daml/blob/9f9d2d0458013cb8f75dfc5c630a1b711c38359a/ledger/ledger-api-common/src/main/scala/com/digitalasset/platform/server/api/services/grpc/GrpcTransactionService.scala#L41